site/cypress/e2e/cypress-feature/submission.spec.js

```diff
@@ -0,0 +1,86 @@
+function setupTestStart() {
```

Comment:
Why does this need to be in a function?  It would be more clear to simply put it in the `beforeEach()`.